doc/plugin_server_nodeattestor_x509pop.md

```diff
@@ -19,6 +19,7 @@ spiffe://<trust domain>/spire/agent/x509pop/<fingerprint>
 | Configuration | Description | Default                 |
 | ------------- | ----------- | ----------------------- |
 | `ca_bundle_path` | The path to the trusted CA bundle on disk. The file must contain one or more PEM blocks forming the set of trusted root CA's for chain-of-trust verification. | |
+| `ca_bundle_paths` | A list of paths to trusted CA bundles on disk. The files must contain one or more PEM blocks forming the set of trusted root CA's for chain-of-trust verification. | |
```

Comment:
If you take my suggestion above (i.e. "If you have more than one file, use `ca_bundle_paths` instead."), then can you update `ca_bundle_path` description with that?